starwars/src/components/StarCard.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./StarWars.css";
+
+const StarCard = props => {
+  return (
```

Comment:
consider using object destructuring before the return statement to help dry up code. `{name, heigh, eight_color, ...rest} = props.charData`